blazor/visual-studio-integration/download-and-installation.md

```diff
@@ -13,22 +13,22 @@ Syncfusion publishes the Visual Studio extension in the below Visual Studio mark
 
 [Visual Studio 2022](https://marketplace.visualstudio.com/items?itemName=SyncfusionInc.BlazorVSExtension)
 
-[Visual Studio 2019](https://marketplace.visualstudio.com/items?itemName=SyncfusionInc.Blazor-Extension) 
+[Visual Studio 2019](https://marketplace.visualstudio.com/items?itemName=SyncfusionInc.Blazor-Extension)
 
 
 ## Prerequisites
 
 The following software prerequisites must be installed to install the Syncfusion Blazor extension, as well as to creating, adding snippet, converting, and upgrading Syncfusion Blazor applications.
 
-* [Visual Studio 2019 16.3 or later](https://visualstudio.microsoft.com/downloads/).
+* [Visual Studio 2022 or later](https://visualstudio.microsoft.com/downloads/).

```

Comment:
remove "or later" in all places

Reply:
Removed " or later" in existing content.